lib/data/datasources/category/category_local_datasource.dart

```diff
@@ -13,13 +14,15 @@ abstract class CategoryLocalDataSource {
     TransactionType type,
     int userId, {
     String? description,
+    Media? media,
   });
   Future<Category> updateCategory(
     String clientId, {
     String? name,
     String? slug,
     TransactionType? type,
     String? description,
+    Media? media,
```

Comment:
```suggestion
```
The goal is that backend and frontend models should be the same? categories don't have media property.